src/editors/containers/ProblemEditor/data/ReactStateOLXParser.js

```diff
@@ -4,6 +4,10 @@ import { ProblemTypeKeys } from '../../../data/constants/problem';
 
 class ReactStateOLXParser {
   constructor(problemState) {
+    const parserOptions = {
+      ignoreAttributes: false,
+      alwaysCreateTextNode: true,
```

Comment:
What would require the parser & builder to have different sets of options here? builderOptions and parserOptions are slightly different.

Reply:
I am not sure. I based this on how options are separated in `OLXParser`

Reply:
Ok i think I understand.